x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/views/curations.tsx

```diff
@@ -0,0 +1,178 @@
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+
+import { useValues, useActions } from 'kea';
+
+import {
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiPageContent,
+  EuiTitle,
+  EuiBasicTable,
+  EuiBasicTableColumn,
+  EuiEmptyPrompt,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { FlashMessages } from '../../../../shared/flash_messages';
+import { KibanaLogic } from '../../../../shared/kibana';
+import { Loading } from '../../../../shared/loading';
+import { EuiButtonTo, EuiLinkTo } from '../../../../shared/react_router_helpers';
+import { ENGINE_CURATIONS_NEW_PATH, ENGINE_CURATION_PATH } from '../../../routes';
+import { generateEnginePath } from '../../engine';
+
+import { CURATIONS_OVERVIEW_TITLE, CREATE_NEW_CURATION_TITLE } from '../constants';
+import { CurationsLogic } from '../curations_logic';
+import { Curation } from '../types';
+
+export const Curations: React.FC = () => {
+  const { dataLoading, curations, meta } = useValues(CurationsLogic);
+  const { loadCurations } = useActions(CurationsLogic);
+
+  useEffect(() => {
+    loadCurations();
+  }, [meta.page.current]);
+
+  if (dataLoading && !curations.length) return <Loading />;
+
+  return (
+    <>
+      <EuiPageHeader>
+        <EuiPageHeaderSection>
+          <EuiTitle size="l">
+            <h1>{CURATIONS_OVERVIEW_TITLE}</h1>
+          </EuiTitle>
+        </EuiPageHeaderSection>
+        <EuiPageHeaderSection>
+          <EuiButtonTo to={generateEnginePath(ENGINE_CURATIONS_NEW_PATH)} fill>
+            {CREATE_NEW_CURATION_TITLE}
+          </EuiButtonTo>
+        </EuiPageHeaderSection>
+      </EuiPageHeader>
+      <EuiPageContent>
+        <FlashMessages />
+        <CurationsTable />
+      </EuiPageContent>
+    </>
+  );
+};
+
+export const CurationsTable: React.FC = () => {
+  const { dataLoading, curations, meta } = useValues(CurationsLogic);
+  const { onPaginate, deleteCurationSet } = useActions(CurationsLogic);
+
+  const columns: Array<EuiBasicTableColumn<Curation>> = [
+    {
+      field: 'queries',
+      name: i18n.translate(
+        'xpack.enterpriseSearch.appSearch.engine.curations.table.column.queries',
+        { defaultMessage: 'Queries' }
+      ),
+      render: (queries: Curation['queries'], curation: Curation) => (
+        <EuiLinkTo
+          data-test-subj="CurationsTableQueriesLink"
+          to={generateEnginePath(ENGINE_CURATION_PATH, { curationId: curation.id })}
+        >
+          {queries.join(', ')}
+        </EuiLinkTo>
+      ),
+      width: '40%',
+      truncateText: true,
+      mobileOptions: {
+        header: true,
+        // Note: the below props are valid props per https://elastic.github.io/eui/#/tabular-content/tables (Responsive tables), but EUI's types have a bug reporting it as an error
+        // @ts-ignore
+        enlarge: true,
+        width: '100%',
+        truncateText: false,
+      },
+    },
+    {
+      field: 'last_updated',
```

Comment:
Something to note: this field currently imports the date string from Enterprise Search's standalone UI as-is from English with no i18n. I plan on opening a separate follow-up PR after this that:

1. Converts this string into a `Date` parseable class (so that we can feed it into react-intl and provide localized dates), and
2. Adds a FormattedDate/FormattedTime helper for timestamps within tables - we currently have timestamps in the Analytics > Recent Queries table as well and they use completely different time formats, which I want to standardize across our app.

Reply:
Ah, good call